.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: go
+
+os: 
+  - linux
+  - osx
+
+branches:
+  only: 
+    - master
+
+sudo: false
```

Comment:
_If you currently specify `sudo: false` in your `.travis.yml`, we recommend removing that configuration._

https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Reply:
I referred it from Dave Cheney's blog: https://dave.cheney.net/2018/07/16/using-go-modules-with-travis-ci
But the blog is pretty old.